### PR TITLE
[R2] limits

### PR DESCRIPTION
**Pull request by deadlypants1973**
deciding between table or sentences
Screenshot 2022-03-10 at 17 20 08 (2)

### CODE REVIEW

File: content/r2/platform/_index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Platform
+pcx-content-type: navigation
+weight: 2
+meta:
+  title: Platform
```
Comment (lukeed) [inline content/r2/platform/_index.md]: This is only necessary if you want a different meta title (for SEO) than the `title:` value
```suggestion
```

File: content/r2/platform/limits.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: Limits
+pcx-content-type: concept
+meta:
+  title: Limits
```
Comment (lukeed) [inline content/r2/platform/limits.md]:
```suggestion
```

File: content/r2/platform/limits.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+# Limits
+
+## Account plan limits
+
+Account have a maximum limit of 1,000 buckets.
```
Comment (lukeed) [inline content/r2/platform/limits.md]:
```suggestion
Accounts have a maximum limit of 1,000 buckets.
```

File: content/r2/platform/limits.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+## Account plan limits
+
+{{<table-wrap>}}
+
+| Feature                 | Limit                     |
+| ------------------------| ------------------------- |
+| Bucket                  | 1000 buckets per account  |
+| Data storage per bucket | Unlimited                 |
+| Object size             | 50 GB per object          |
```
Comment (sdnts) [inline content/r2/platform/limits.md:16]: I don't think we need the `per object` in here?

File: content/r2/platform/limits.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+{{</table-wrap>}}
+
+To increase these limits, contact your Cloudflare account team.
```
Comment (sdnts) [inline content/r2/platform/limits.md:20]: Did you mean 'contact your Cloudflare account team admin.` perhaps?

File: content/r2/platform/limits.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+# Limits
+
+## Account plan limits
+
+{{<table-wrap>}}
+
+| Feature                 | Limit                     |
+| ------------------------| ------------------------- |
+| Bucket                  | 1000 buckets per account  |
```
Comment (sdnts) [inline content/r2/platform/limits.md:14]: I think we may be able to get rid of `per account` here too

**Review by sdnts — APPROVED** [review]
Left a couple of nitpick comments

File: content/r2/platform/limits.md (hunk 4 above)
Comment (lukeed) [reply content/r2/platform/limits.md:16]: Both are necessary. We've come back thru previous tables in other docs to add the extra clarification due to reader requests.

**Review by lukeed — APPROVED** [review]

File: content/r2/platform/limits.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+---
+title: Limits
+pcx-content-type: concept
+---
```
Comment (marciocloudflare) [inline content/r2/platform/limits.md:4]: Don't we need `weight` there?

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**